modules/core/src/v2/wallet.ts

```diff
@@ -51,6 +51,13 @@ export interface Memo {
     type: string;
 }
 
+/**
+ * We only let a small subset of these params get through in the route.
```

Comment:
nit: This doesn't really inform other developers what this interface does or is to be used for.

- why only a small subset of params?
- what are they?
- when are they likely to be passed?

Reply:
ah yeah that's fair. I can document more clearly what build params can be used here.